logging/monolog_exclude_http_codes.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Logging
+   single: Logging; Exclude HTTP Codes
+   single: Monolog; Exclude HTTP Codes
+
+How to Configure Monolog to Exclude Specific HTTP Codes from the Log
+====================================================================
+
+Sometimes your logs become flooded with unwanted HTTP errors, for example,
+403s and 404s. When using a ``fingers_crossed`` handler, you can exclude
```

Comment:
Quick question: is the mention of the `fingers_crossed` handler just an example or does this feature only work with that specific handler?